co-simulation/bundle/src/assembly/resources/scenarios/Town04/ns3/ns3_federate_config.xml

```diff
@@ -74,7 +74,7 @@
  <default name="ns3::ConstantRateWifiManager::DataMode" value="OfdmRate6Mbps"/>
  <default name="ns3::ConstantRateWifiManager::ControlMode" value="OfdmRate6Mbps"/>
 
- <default name="ns3::QueueBase::MaxPackets" value="400"/>
+ <!-- <default name="ns3::QueueBase::MaxPackets" value="400"/> -->
```

Comment:
I'd recommend adding a comment above that describes why we've removed this parameter for now.